README.md

```diff
@@ -250,10 +250,79 @@ var text2 = Stone.ngettext("{n} apple", "{n} apples", 3, {n: 3});
 var text3 = Stone.ngettext("{n} apple", "{n} apples", 3); // 'n' is an implicit replacement of given number
 ```
 
+
 ### Stone.lazyNgettext
 
 Same as `Stone.ngettext` but returns a `Stone.LazyNString` instead of a `String`.
 
+
+### Stone.pgettext
+
+Translates the given string to the current language using a context argument, helping with string that could be the same in the C
+language but could mean different things in other languages.
+
+    String: Stone.pgettext( <context>, <string> [, locale] );
+    String: Stone.pgettext( <context>, <string> [, replacements] [, locale] );
+
+**params:**
+
+* `context`: The string context.
+* `string`: The string to translate.
+* `locale`: The locale string to use for translation (optional, default: current locale).
+* `replacements`: an object containing replacements for the string (optional, see example below).
+
+**returns:**
+
+The translated string.
+
+**Examples:**
+
+```javascript
+var text1 = Stone.pgettext("going back", "Back");
+var text1 = Stone.pgettext("back of an object", "Back", "it");
```

Comment:
Please add an example with replacements (or change the text line 272 that states there is an example :) )